Feature/split table

Bluesmile82 opened the pull request:
This PR adds the splittedColumns prop to the local Table component that creates two different tables synced by a ScrollSync component (from React virtualized). Also, it applies it to the GHG emissions page table and styles it according to designs.

![table](https://user-images.githubusercontent.com/9701591/78709571-2a0ca500-7914-11ea-9ed2-8102baf2bd7c.gif)

kfilip94 commented:
I can't scroll the table fully to the right side, it clips a few columns there:
[demo link](http://localhost:3000/ghg-emissions?regions=AILAC%2CBRICS%2CEAP%2CECA%2CBRA%2CCHN%2CIND%2CRUS%2CZAF%2CTOP&source=17)
![image](https://user-images.githubusercontent.com/15097138/78800907-ec5f5900-79b3-11ea-83ba-1d1987d98606.png)

kfilip94 commented:
- Show data by -> `Regions-Per GDP` - the unit doesn't fit the row, not sure if this is an old issue or related with these table changes:
![image](https://user-images.githubusercontent.com/15097138/78801858-15341e00-79b5-11ea-947f-60fda50bb3a8.png)

- Sometimes the columns are too narrow for greater values:
[demo link](http://localhost:3000/ghg-emissions?breakBy=gas&chartType=line&gases=59%2C60%2C61%2C63%2C62&regions=TOP&source=17)
![image](https://user-images.githubusercontent.com/15097138/78802670-00a45580-79b6-11ea-9643-9a035b7d5251.png)

- It doesn't look good on the mobiles, but I assume we can live with that for now?:
![image](https://user-images.githubusercontent.com/15097138/78801396-90490480-79b4-11ea-84b0-f8d2ba64f570.png)

Bluesmile82 commented:
Ok, this needed a couple of fixes. Thanks for the comments. I couldn't reproduce this one 'Sometimes the columns are too narrow for greater values:' Could you check it and if it fails give me the exact parameters? The ids are not the same in different environments (This is a bug we have to fix)

kfilip94 commented:
Hey, try these filters:
![image](https://user-images.githubusercontent.com/15097138/78818032-ee351680-79cb-11ea-9974-fe6441bf6ab9.png)

Bluesmile82 commented:
I updated this PR adding:

- Fix for the border style
- Fix minimum width for every column so the numbers don't get cut
- Added links for the countries and EUU leading to the country page

Ready again!

kfilip94 commented on app/javascript/app/components/ghg-emissions/ghg-emissions-selectors/ghg-emissions-selectors-data.js:
```diff
@@ -503,8 +508,31 @@ export const getTableData = createSelector(
-    return pivot;
+export const getTitleLinks = createSelector(
+  [getTableData, getModelSelected, getRegions, getCountries],
+  (data, model, regions, countries) => {
```
Both selectors `getRegions` and `getCountries` can return `null`, could you add guards for `regions` and `countries` here?
